docs/source/design/control_plane.rst

```diff
@@ -12,7 +12,7 @@ backend set up and have access to it (a :std:ref:`local sandbox <flyte:deploymen
 
 The video below features a demo of FlyteRemote, followed by all relevant information to create, fetch and execute FlyteRemote objects.
 
-..  youtube:: Vjukdg8LIh8
+..  youtube:: 1Dbj3t_1Ltg
```

Comment:
this video seems to be private - https://www.youtube.com/watch?v=1Dbj3t_1Ltg

Reply:
It is now public.

Reply:
It is now public.